AirCasting/Graph/Graph.swift

```diff
@@ -64,7 +64,7 @@ struct Graph: UIViewRepresentable {
         let uiView = AirCastingGraph(onDateRangeChange: { newRange in
             rangeChangeAction?(newRange)
         })
-        try? uiView.updateWithThreshold(thresholdValues: thresholds.rawThresholdsBinding.wrappedValue)
+        try? uiView.updateWithThreshold(thresholdValues: stream.isTemperature && userSettings.convertToCelsius ? thresholds.rawThresholdsBindingCelsius.wrappedValue : thresholds.rawThresholdsBinding.wrappedValue)
```

Comment:
Here maybe we could use only var to make it clearer?

` var properThresholds: [Float] {
        stream.isTemperature && userSettings.convertToCelsius ? thresholds.rawThresholdsBindingCelsius.wrappedValue : thresholds.rawThresholdsBinding.wrappedValue
    }`